components/logins/src/login.rs

```diff
@@ -2,6 +2,135 @@
+//!                     If present, the login should only be used in forms with a corresponding field.
+//!                     There are no restrictions on the contents of this string.
+//!
+//!                 XXX TODO: can this be present if `formSubmitURL` is not set?
+//!
+//!  -`passwordField`:  The HTML form field into which the `password` field should be filled, if any.
+//!                     If present, the login should only be used in forms with a corresponding field.
+//!                     There are no restrictions on the contents of this string.
+//!
+//!                 XXX TODO: can this be present if `formSubmitURL` is not set?
+//!
+//!  - `timesUsed`: A lower bound on the number of times this record as been "used", as an integer.
+//!                 This number may not record uses that occurred on other devices, since some legacy
+//!                 sync clients do not record this information. It may be zero for records obtained
+//!                 via sync that have never been used locally.
+//!
+//!                 This field is managed internally by the logins store and any attempt by the
+//!                 application to specify a value will be ignored.
+//!
+//!                 XXX TODO: are these local machine timestamps, and thus possibly wildly inaccurate?
+//!                 XXX TODO: should we talk about merge strategies here?
+//!
+//!  - `timeCreated`: An upper bound on the time of creation of this login, in integer milliseconds from the unix epoch.
+//!                   This is an upper bound because some legacy sync clients do not record this information;
+//!                   in that case newer clients set `timeCreated` when they see the record for the first time.
+//!
+//!                 This field is managed internally by the logins store and any attempt by the
+//!                 application to specify a value will be ignored.
+//!
+//!                 XXX TODO: are these local machine timestamps, and thus possibly wildly inaccurate?
+//!                 XXX TODO: should we talk about merge strategies here?
+//!
+//!  - `timeLastUsed`: A lower bound on the time of last use of this login, in integer milliseconds from the unix epoch.
+//!                    This is a lower bound because some legacy sync clients do not record this information;
+//!                    in that case newer clients set `timeLastUsed` when they use the record for the first time.
+//!
+//!                 This field is managed internally by the logins store and any attempt by the
+//!                 application to specify a value will be ignored.
+//!
+//!                 XXX TODO: are these local machine timestamps, and thus possibly wildly inaccurate?
+//!                 XXX TODO: should we talk about merge strategies here?
+//!
+//!  - `timePasswordChanged`: A lower bound on the time that the `password` field was last changed, in integer milliseconds from the unix epoch.
```

Comment:
Maybe we should be explicit here and say that username (or other field) changes shouldn't be recorded here as that's what we want and it's been discussed a few times.

Reply:
For more context, this property is useful for breach alerts of passwords since changing the username wouldn't have addressed the breach.